backend/account/views.py

```diff
@@ -1,3 +1,22 @@
-# from django.shortcuts import render
+from django.http import JsonResponse, HttpResponseBadRequest
+from django.views.decorators.http import require_http_methods
 
-# Create your views here.
+import json
+from json.decoder import JSONDecodeError
+
+from .models import User
+
+@require_http_methods(["POST"])
+def signup(request):
+    try:
+        body = request.body.decode()
+        email = json.loads(body)['email']
+        username = json.loads(body)['username']
+        password = json.loads(body)['password']
+    except (KeyError, JSONDecodeError):
+        return HttpResponseBadRequest()
+    user = User.objects.create_user(email=email, username=username)
+    user.set_password(password)
```

Comment:
Is there any reason you implemented set_passwords instead of writing User.objects.create_user(username=username, email=email, password=password)?

Reply:
This is for security. If you pass password like you said, the raw password is saved on DB. Instead, set_password function generates hashed password and saves it on DB.

Reply:
Thank you for the comment!